keyboards/ropro/keymaps/default/keymap.c

```diff
@@ -0,0 +1,84 @@
+Hey /* Copyright 2015-2017 Jack Humbert
```

Comment:
Typo. Also, realistically, your name should be on this file.

```suggestion
/* Copyright 2019 Garret G. (TheRoyalSweatshirt)
```

Use whichever variant of your name you prefer (real, reddit, both, etc.).